pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/Policies.java

```diff
@@ -100,11 +100,10 @@
 
     @SuppressWarnings("checkstyle:MemberName")
     @Deprecated
-    public SchemaAutoUpdateCompatibilityStrategy schema_auto_update_compatibility_strategy =
-        SchemaAutoUpdateCompatibilityStrategy.Full;
+    public SchemaAutoUpdateCompatibilityStrategy schema_auto_update_compatibility_strategy = null;
 
     @SuppressWarnings("checkstyle:MemberName")
-    public SchemaCompatibilityStrategy schema_compatibility_strategy = SchemaCompatibilityStrategy.UNDEFINED;
+    public SchemaCompatibilityStrategy schema_compatibility_strategy = null;
```

Comment:
Get Schema compatibility strategy of namespace API has been the break that doesn't read broker level, so I think we can set `null` as  `schema_compatibility_strategy` value. When `schema_auto_update_compatibility_strategy` will deleted, this API still returns `null`.